census/src/main/java/io/grpc/census/CensusStatsModule.java

```diff
@@ -787,6 +799,8 @@ public ServerStreamTracer newServerStreamTracer(String fullMethodName, Metadata
           tagger
               .toBuilder(parentCtx)
               .putLocal(RpcMeasureConstants.GRPC_SERVER_METHOD, methodTag)
+              .putLocal(INSTRUMENTATION_SOURCE, LIBRARY_NAME)
```

Comment:
From the looks of it, we should just do something like this on client-side as well and not update the individual record sites.